FESS-GUI/mainwindow.cpp

```diff
@@ -609,3 +666,18 @@ void MainWindow::on_lineEditPassword_textEdited(const QString &password)
 }
 
 
+
+void MainWindow::on_actionLock_frame_rate_at_30FPS_triggered(bool checked)
+{
+    dataTimer->stop();
+    if(checked)
+        refreshRate = 30;
```

Comment:
a refreshRate of 30 is a refresh ever 30 ms, not 30 refreshes per second. 1000(ms / sec) / 30 (fps) = 33.333 (ms/frame). So the refresh rate should be 33